packages/components/src/sortTabs/SortTab.js

```diff
@@ -0,0 +1,153 @@
+  };
+
+  render() {
+    const { label, first, last, active, icon } = this.props;
+    const { hovered } = this.state;
+    return (
+      <Hoverable
+        onMouseEnter={this.handleOnMouseEnter}
+        onMouseLeave={this.handleOnMouseLeave}
+      >
+        <TouchableWithoutFeedback onPress={this.handleOnPress}>
+          <View
+            style={[
+              styles.container,
+              first && styles.containerFirst,
+              last && styles.containerLast,
+              active && styles.containerActive,
+              hovered && styles.containerHovered,
+            ]}
+          >
+            <View>
+              <Text
+                style={[styles.label, active && styles.labelActive]}
+                align="left"
+                numberOfLines={1}
+              >
+                {label}
+              </Text>
+              <View style={styles.priceDurationContainer}>
+                <Text style={[styles.label]} align="left" numberOfLines={1}>
+                  1,394 Kč
+                </Text>
+                <Text
+                  style={[styles.durationLabel]}
+                  align="left"
+                  numberOfLines={1}
+                >
+                  9h 3m
+                </Text>
+              </View>
+            </View>
+            <Icon
+              size="small"
+              name={icon}
+              color={
+                active
+                  ? defaultTokens.paletteProductNormal
+                  : defaultTokens.colorIconPrimary
+              }
+            />
+          </View>
+        </TouchableWithoutFeedback>
+      </Hoverable>
+    );
+  }
+}
+
+const borderRadius = parseInt(defaultTokens.borderRadiusSmall, 10);
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    padding: parseInt(defaultTokens.paddingButtonSmall, 10),
+    backgroundColor: defaultTokens.paletteWhite,
+    overflow: 'hidden',
+    web: {
+      userSelect: 'none',
+    },
+  },
+  containerFirst: {
+    borderTopStartRadius: borderRadius,
+    borderBottomStartRadius: borderRadius,
+  },
+  containerLast: {
+    borderTopEndRadius: borderRadius,
+    borderBottomEndRadius: borderRadius,
+    marginEnd: 0,
+  },
+  containerActive: {
+    backgroundColor: defaultTokens.paletteWhite,
+    borderBottomWidth: 2,
+    borderBottomColor: defaultTokens.paletteProductNormal,
+  },
+  containerHovered: {
+    backgroundColor: defaultTokens.paletteWhiteHover,
+  },
+  priceDurationContainer: {
+    paddingTop: parseInt(defaultTokens.paddingTag, 10),
+    flex: 1,
+    flexDirection: 'row',
+  },
+  durationLabel: {
+    color: defaultTokens.paletteInkLight,
+    fontWeight: 'normal',
+    marginStart: parseInt(defaultTokens.paddingButtonSmall, 10),
+    fontSize: parseInt(defaultTokens.fontSizeTextSmall, 10),
+  },
+  label: {
+    flex: 1,
+    color: defaultTokens.paletteInkDark,
+    fontWeight: '500',
```

Comment:
You can use `designTokens.fontWeightMedium` from `@kiwicom/orbit-design-tokens` for this one.

Reply:
Done!